data/elevation-and-terrain/exploring-lidar/index.html

```diff
@@ -62,9 +62,10 @@ <h4>Lidar Resources</h4>
   <li><a href="{% link data/elevation-and-terrain/index.html %}">AGRC Elevaton and Terrain Data</a></li>
   <li><a href="{% link _posts/2017-11-27-dumbing-down-terrain-with-contour-polygons.md %}">Dumbing Down Terrain with Contour Polygons</a></li>
   <li><a href="https://docs.google.com/document/d/188M11SoXovnHeg3J8RwEb89FsWBJQJEwFB5SWyXwask/edit?usp=sharing">Working with Mosaic and Reference Datasets</a></li>
+  <li><a href="https://viewer.nationalmap.gov/basic/">The National Map</a></li>
+  <li><a href="http://www.opentopography.org/index.php">Open Topography</a></li>
```

Comment:
**issue**: use https version without www since they redirect
```suggestion
  <li><a href="https://opentopography.org/index.php">Open Topography</a></li>
```